src/main/kotlin/no/nav/tiltakspenger/arena/repository/ArenaKodeverk.kt

```diff
@@ -0,0 +1,138 @@
+package no.nav.tiltakspenger.arena.repository
+
+enum class ArenaSakStatus(val navn: String) {
+    AKTIV("Aktiv"),
+    AVSLU("Lukket"),
+    INAKT("Inaktiv"),
+}
```

Comment:
Ser at alle enumene i denne fila likner veldig på enumer i fila `YtelseSakDTO`.
Er de like? Skal vi isåfall slette de som finnes i `YtelseSakDTO`, og holde denne nye fila som 'master'?

Reply:
Det er et godt spørsmål.. Vi har de samme enumene nå i to kontekster; fra db og fra webservice. Det er kanskje litt unødvendig. Men mtp at dagpenger potensielt skal kopiere koden vår, så er det kanskje greit sånn i første omgang, så blir det litt mer "selvstående" og lettere å kopiere. Eller hva tror du?